content/en/docs/release-notes/release-notes-1.3.md

```diff
@@ -5,11 +5,23 @@ weight: 800
 type: "docs"
 ---
 
+This release prepares for the adoption of the upstream Kubernetes CSR API and improves interopability with Hashicorp Vault enterprise.
+A slew of bugs have also been squashed.
+
+Special thanks to the external contributors whose PRs landed in this release:
+
+* [@teejaded](https://github.com/teejaded)
+* [@7opf](https://github.com/7opf)
+* [@yann-soubeyrand](https://github.com/yann-soubeyrand)
+* [@Kirill-Garbar](https://github.com/Kirill-Garbar)
+* [@joshuastern](https://github.com/joshuastern)
+* [@lalitadithya](https://github.com/lalitadithya)
+
 Please read the [upgrade notes](/docs/installation/upgrading/upgrading-1.2-1.3/) before upgrading.
 
-Aside from that, there have been numerous bug fixes and features summarized below.
+As always, the full changelog is available on the [GitHub release](https://github.com/jetstack/cert-manager/releases/tag/v1.3.0)
 
-# Breaking Changes
+# Deprecated Features and Breaking Changes
```

Comment:
Is it in this release that we made CR immutable? @JoshVanL ? If so, add a note about that.